apis/v1alpha2/validation/validation_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	gatewayv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+)
+
+func pathMatchTypePtr(s string) *gatewayv1a2.PathMatchType {
```

Comment:
the file provides test utils. Maybe should be named as such? if these func (e.g. toPtr) are commonly used even in validation path then maybe create a api util package (with its own tests) then import it in validation and validation execution paths.